net-libs/mbedtls/mbedtls-2.6.0.recipe

```diff
@@ -0,0 +1,118 @@
+SUMMARY="An open source, portable, easy to use, readable and flexible SSL library"
```

Comment:
The SUMMARY line is a bit long (>70 chars). What do you think of:
```
SUMMARY="A lightweight crypto and SSL/TLS library with command line tools"
```
Hmm, I'm not really satisfied with this either, so if anyone has a better suggestion ;-)

Reply:
If you remove `An `, it becomes 69 characters :)